tools/build/golang/Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN go get -tags netgo \
 		github.com/gogo/protobuf/gogoproto \
 		github.com/gogo/protobuf/protoc-gen-gogoslick \
 		github.com/golang/dep/... \
-		github.com/golang/lint/golint \
+		golang.org/x/lint/golint \
```

Comment:
The whole `tools` directory is actually a copy of https://github.com/weaveworks/build-tools so cannot be changed here.